packages/terra-form-radio/src/RadioField.jsx

```diff
@@ -138,7 +138,7 @@ const RadioField = (props) => {
   const content = React.Children.map(children, (child) => {
     if (child && child.type.isRadio) {
       return React.cloneElement(child, {
-        inputAttrs: { 'aria-describedby': ariaDescriptionIds },
+        inputAttrs: { 'aria-describedby': ariaDescriptionIds, ...child.props.inputAttrs },
```

Comment:
```suggestion
        inputAttrs: { ...child.props.inputAttrs, 'aria-describedby': ariaDescriptionIds },
```
This prevents 'aria-describedby' from getting overridden by consumer props which would lead to unexpected Accessibility responses.